PaulasWeatherApp/PaulasWeatherApp/Model/Singleton.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Singleton.swift
+//  PaulasWeatherApp
+//
+//  Created by Stef, Paula on 8/1/22.
+//
+
+import Foundation
+import UIKit
+
+class Singleton: NSObject {
+    
+    static let sharedInstance = Singleton()
```

Comment:
Singletons should be avoided https://www.swiftbysundell.com/articles/avoiding-singletons-in-swift/